diaries/diary_TiffanyAH KING.tex

```diff
@@ -172,7 +172,46 @@ \section{Iteration 1}
 
 
 
-%\section{Iteration 2}
+\section{Iteration 2}
```

Comment:
that is strange it is already on master see: https://github.com/binarytrails/kakuro/commit/a26843f41e33c9e99ada0b5c32e45b9dff65bbed